docs/product/insights/mobile-vitals/screen-loads.mdx

```diff
@@ -17,13 +17,13 @@ Sentry tracks TTID automatically, but [TTFD](/product/insights/mobile-vitals/#ti
 **For Android:**
 
 - `>=5.0` for automatic activity transactions
-- `>=6.10.0` for TTID
-- `>=6.17.0` for TTFD, (released March, 2023)
+- `>=6.10.0` for [TTID](/platforms/android/tracing/instrumentation/automatic-instrumentation/#time-to-initial-display)
+- `>=6.17.0` for [TTFD](/platforms/android/tracing/instrumentation/automatic-instrumentation/#time-to-full-display), (released March, 2023)
```

Comment:
```suggestion
- `>=6.17.0` for [TTFD](/platforms/android/tracing/instrumentation/automatic-instrumentation/#time-to-full-display)
```

Can we removed the `(released <month> <year>)` notes for all the bullets in this section (for flutter & react too)? I feel like it doesn't add anything at this point